eclipse-theia/theia-blueprint
# Tests and Initial Renaming to Theia-Blueprint

#### What it does
* Add more information to README
* start rename process from theia-example to theia-blueprint. Missing are places where the buildprocess needs to be adapted. This will be done in a smaller follow up, because the CI-pipeline is more difficult in this case. 
* add basic (smoke) tests that checks extensions view and title

#23 and #25 are not yet fully addressed with this commit
closes #6

Contributed on behalf of STMicroelectronics

#### How to test
* run the tests as described in readme
* build application and check title / filename

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
